examples/info-operator-service/examples/get_operator_info.rs

```diff
@@ -94,11 +94,11 @@ pub async fn register_operator(pvt_key: &str, bls_key: &str, http_endpoint: &str
 
     let el_chain_reader = ELChainReader::new(
         get_test_logger(),
-        Address::ZERO,
+        Some(Address::ZERO),
```

Comment:
This should be `None`

Reply:
added None. Users could be confused with using ZERO maybe?

Reply:
Yeah. I was thinking maybe using `Option<NonZeroAddress>` would be a better alternative, but that type doesn't exist 😅